tools/MultiProjectValidator/ProjectLoader.cs

```diff
@@ -29,10 +30,23 @@ private static List<ProjectContext> LoadProjectContexts(string[] projectFiles)
             {
                 var fileTargetContexts = ProjectContext.CreateContextForEachTarget(file);
 
-                projectContexts.AddRange(fileTargetContexts);    
+                foreach(var context in fileTargetContexts)
+                {
+                    if (!TargetIsDesktop(context))
+                    {
+                        projectContexts.Add(context);
+                    }
+                }  
             }
 
             return projectContexts;
         }
+
+        private static bool TargetIsDesktop(ProjectContext context)
```

Comment:
both? The rule should decide if there are contexts it doesn't care about, but something tied to Context should probably determine if that context is desktop...